octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po

```diff
@@ -1324,39 +1347,65 @@ msgstr ""
 "sie wieder startet."
 
 #: octoprint_mrbeam/static/js/user_notification_viewmodel.js:55
+msgid "No update information"
+msgstr ""
```

Comment:
Why are these added here in this ticket?

Reply:
Good catch, I will revert the translation and add it to a new ticket